src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -83,10 +83,9 @@ function ReceiptSelector(props) {
 
     const iouType = lodashGet(props.route, 'params.iouType', '');
     const reportID = lodashGet(props.route, 'params.reportID', '');
+    const pageIndex = lodashGet(props.route, 'params.pageIndex', 1);
```

Comment:
hm, I understand the edge case you are describing. you are saying:

> This is likely to happen on native devices

But I think it's unlikely it will happen? I'd even say I'd never happen, because this code is in index.**native**.js, right?

Reply:
And on native you can't temper the URL